### DISCUSSION TITLE
Processes visuals

### DISCUSSION THREAD

**Discussion by helixphoenix** [kiali/kiali#3744] [General]
Actually I have been using weave scope previously and maybe it can be inspiration. Its resource consumption was very high but if we could have lightweight version of  it where actually we can show dependencies and processes actually we can have an idea of how our system looks which helps to understand it better.

**Comment by jshaughn** [comment]
@phoenitunes So, can you list some more explicit features you think could benefit the Kiali user?  In last week's workshop I remember people asking whether Kiali could, for example, restart pods on demand (and not just as a result of, say, adding injection to a workload).   But this discussion is maybe suggesting the ability to see a graph based on config as opposed to traffic.    Maybe you could add a little more detail around the use case you are thinking about.
